prime-router/docs/simplereport-data-dictionary.md

```diff
@@ -0,0 +1,224 @@
+# Data Dictionary for data incoming from SimpleReport
```

Comment:
I'd much prefer to auto-generate this, but this took me just a few minutes to do it manually, so I'm throwing this out there for now.